Monika-After-Story/MonikaModDev
# NOU outfit

### TODO:
- [x] thumbs
- [x] verify looks good
- [x] unlock event

### Credits:
Eroneko, commissioned by Finikyu

### Changes:
- new event
- new event outfit
- new pm var

### Testing:
- verify the event condition is valid and you can unlock the outfit
- verify the expressions and writing are good
- try different paths in the monilogue
- checkout the new pm var
- verify
- - before the event
- - - the outfit is locked
- - - you're not eligible before the condition is met, can't see the event
- - after the event
- - - the outfit remains unlocked 
- - - the event is locked
